### PR TITLE
Iss995: refactor southwest profile into regular profile

### LINKED ISSUE #995: Refactor profileSouthwest into profile
Ideally, these should not be different templates.

### PR DESCRIPTION
**Pull request by JRustyHaner**
fixes #995

### CODE REVIEW

File: mofacts/client/lib/router.js [hunk 1]
```diff
@@ -247,7 +246,7 @@ Router.route('/profile', {
       if (loginMode === 'southwest') {
         console.log('southwest login, routing to southwest profile');
         Session.set('curModule', 'profileSouthwest');
-        this.redirect('/profileSouthwest');
```
Comment (MegaGeese) [inline mofacts/client/lib/router.js:250]: redirecting to the route will cause iron router to not render the profile page.

**Review by MegaGeese — CHANGES_REQUESTED** [review]

File: mofacts/client/lib/router.js (hunk 1 above)
Comment (JRustyHaner) [reply mofacts/client/lib/router.js:250]: this isn't true, tested it with live data.

Comment (JRustyHaner) [reply mofacts/client/lib/router.js:250]: Also, did you mean to comment on the existing code?

Comment (MegaGeese) [reply mofacts/client/lib/router.js:250]: meant to comment on the new code

Comment (MegaGeese) [reply mofacts/client/lib/router.js:250]: please test again with the signinsouthwest route.

File: mofacts/client/views/home/profile.js [hunk 2]
```diff
@@ -294,7 +294,12 @@ Session.set('speechAPIKey', null);
 Template.profile.rendered = async function() {
   sessionCleanUp();
   Session.set('showSpeechAPISetup', true);
-  allTdfs = await meteorCallAsync('getAllTdfs');
+  if(Meteor.user().profile.loginMode === 'southwest') {
+    const curSectionId = Meteor.user().profile.curClass.sectionId;
+    const allTdfs = await meteorCallAsync('getTdfsAssignedToStudent', Meteor.userId(), curSectionId);
```
Comment (MegaGeese) [inline mofacts/client/views/home/profile.js:299]: This variable would be out of scope for the rest of the function throwing errors and breaking profile

**Review by MegaGeese — CHANGES_REQUESTED** [review]

File: mofacts/client/lib/router.js (hunk 1 above)
Comment (JRustyHaner) [reply mofacts/client/lib/router.js:250]: worked fine still, but I will change it to a render.

File: mofacts/client/views/home/profile.js (hunk 2 above)
Comment (JRustyHaner) [reply mofacts/client/views/home/profile.js]: true

**Pull request merged**